classes/mender-sdimg.bbclass

```diff
@@ -167,15 +167,18 @@ EOF
     # Call WIC
     IMAGE_CMD_wic
 
+    wicimgname="${IMGDEPLOYDIR}/${IMAGE_NAME}${IMAGE_NAME_SUFFIX}.wic"
```

Comment:
Is `IMGDEPLOYDIR` documented anywhere? I can't find it in the Yocto documentation. Is it one of the "secret" variables that wic uses?